src/devtools/client/debugger/src/components/Editor/LineHitCounts.tsx

```diff
@@ -138,7 +146,7 @@ function LineHitCounts({ editor, isCollapsed, setIsCollapsed }: Props) {
       doc.off("change", drawLines);
       doc.off("swapDoc", drawLines);
     };
-  }, [editor, hitCountMap, isCollapsed, minHitCount, maxHitCount, setIsCollapsed]);
+  }, [editor, hitCountMap, isCollapsed, minHitCount, maxHitCount, updateHitCountsMode]);
```

Comment:
This will break memoization, because our `useStringPref` always returns a new inline function for `update` 😢 We should fix that to be memoized.